### ISSUE TITLE
Reporting with a block pop density focus

### ISSUE THREAD

**Issue by antrim** [pouyalireza/TNAST_MAVEN#26]
Reporting with a block pop density focus [Trillium leads this in consultation with OSU]

1. For all reports with a stop radius parameter add a measure of pop density (pop per land area) based on blocks within specified block buffer. 
2. Count block density only once.
3. Is block area available from census or do we need to calculate?
4. Any significant performance issues?
5. If this data is new to TNA tool this might require touching a whole chain of program functionality from data import to UI and data export and everything in between.
6. Related suggestions? From TNA tool final report document, it looks like 0 pop blocks are currently being dropped from db.

**Comment by antrim** [comment]
Duplicate of #17. Closing
